src/iop/colorout.c

```diff
@@ -475,39 +478,34 @@ void process_sse2(struct dt_iop_module_t *self, dt_dev_pixelpipe_iop_t *piece, c
   const dt_iop_colorout_data_t *const d = (dt_iop_colorout_data_t *)piece->data;
   const int ch = piece->colors;
   const int gamutcheck = (d->mode == DT_PROFILE_GAMUTCHECK);
+  const size_t npixels = (size_t)roi_out->width * roi_out->height;
+  float *const restrict out = (float *)ovoid;
 
   if(d->type == DT_COLORSPACE_LAB)
   {
     dt_iop_image_copy_by_size(ovoid, ivoid, roi_out->width, roi_out->height, ch);
   }
   else if(!isnan(d->cmatrix[0]))
   {
-// fprintf(stderr,"Using cmatrix codepath\n");
+    const float *const restrict in = (const float *const)ivoid;
+    const float *const restrict cmatrix = d->cmatrix; 
+    const __m128 m0 = _mm_set_ps(0.0f, cmatrix[6], cmatrix[3], cmatrix[0]);
+    const __m128 m1 = _mm_set_ps(0.0f, cmatrix[7], cmatrix[4], cmatrix[1]);
+    const __m128 m2 = _mm_set_ps(0.0f, cmatrix[8], cmatrix[5], cmatrix[2]);
+ fprintf(stderr,"Using cmatrix codepath\n");
```

Comment:
You've missed the spot ;) darktable now prints for me "Using cmatrix codepath" constantly ;)

Reply:
Fixed in master.